src/sage/topology/cell_complex.py

```diff
@@ -165,7 +165,7 @@ def dimension(self):
             4
         """
         try:
-            return max([x.dimension() for x in self._facets])
+            return max([-2] + [x.dimension() for x in self._facets])
         except AttributeError:
```

Comment:
this could use `max(iterator, default=-2)`

Reply:
Thanks; that's better.